web/ts/chat/Chat.ts

```diff
@@ -379,6 +406,7 @@ type ChatMessage = {
     visible: true;
     deleted: boolean;
     isGrayedOut: boolean;
+    renderVersion: number;
```

Comment:
what exactly is a renderVersion? I can only find it being `0` here, could we use a boolean?

Reply:
The issue is that alpine.js was not rerendering the messages for admin layout as the key was not changing after a patch. To indicate that the there is a different version of the message now, I increment the renderVersion if a message gets patched, which already exists. So theoretically later on it would support multiple patches of a message with corresponding layout update. A boolean could work too, by toggling it between true and false but this would be difficult to understand I think.